src/sentry/workflow_engine/handlers/detector/stateful.py

```diff
@@ -298,6 +299,14 @@ def get_state_data(
             )
         return results
 
+    def get_all_triggered_group_keys(self) -> list[DetectorGroupKey]:
+        """Get all group keys that are currently triggered for this detector"""
+
+        triggered_states = self.detector.detectorstate_set.filter(is_triggered=True).exclude(
+            state=DetectorPriorityLevel.OK.value
+        )
+        return [state.detector_group_key for state in triggered_states]
```

Comment:
🤔 i think we might want to have this work without state, so we can have the method live on the base detector. (since that's where the grouping info was added, it seems like we should support this w/o detector state)